pandas/tests/frame/test_constructors.py

```diff
@@ -287,8 +287,49 @@ def test_constructor_dict(self):
         with tm.assert_raises_regex(ValueError, msg):
             DataFrame({'a': 0.7}, columns=['a'])
 
-        with tm.assert_raises_regex(ValueError, msg):
```

Comment:
make a separate test (this change), with a comment

Reply:
(done)